package-lock.json

```diff
@@ -0,0 +1,13 @@
+{
+    "name": "tick",
+    "version": "1.0.0",
```

Comment:
this is only used to run tests with node, maybe it's not necessary to keep in git, and can just be added to .gitignore maybe.